lms/templates/certificates/_accomplishment-banner.html

```diff
@@ -61,7 +62,8 @@ <h3 class="sr-only">${_("Print or share your certificate:")}</h3>
                       <button data-tooltip="${_('Share on Twitter')}"
                         class="action action-share-twitter btn-inverse btn-small icon-only"
                         title="${_('Share on Twitter')}"
-                        onclick="popupWindow('${twitter_url}', 'tweetWindow', 640, 480); return false;">
+                        ## xss-lint: disable=mako-invalid-html-filter
```

Comment:
Please include the comment explaining why the linter is being disabled, as suggested by Robert.
`## xss-lint: disable=mako-invalid-html-filter  Linter mistakes onclick as html.`